app/javascript/components/map/components/basemaps/basemaps-component.jsx

```diff
@@ -0,0 +1,177 @@
+    const { selectBasemap } = this.props;
+    return (
+      <button
+        className="basemaps-list-item-button"
+        onClick={() => selectBasemap(item)}
+      >
+        <div
+          className="basemaps-list-item-image"
+          style={{
+            backgroundImage: `url(/assets/basemaps/${item.id}.png)`
+          }}
+        />
+        <p className="basemaps-list-item-name">{item.label}</p>
+      </button>
+    );
+  }
+
+  renderDropdownBasemap(item) {
+    const { selectBasemap, activeBasemap } = this.props;
+    const { landsatYears } = this.state;
+    const year = activeBasemap.year || landsatYears[0].value;
+
+    return (
+      <button
+        className="basemaps-list-item-button"
+        onClick={() => selectBasemap(basemaps.landsat, year)}
+      >
+        <div
+          className="basemaps-list-item-image"
+          style={{
+            backgroundImage: `url(/assets/basemaps/${item.id}.png)`
+          }}
+        />
+        <span
+          className="basemaps-list-item-name"
+          onClick={e => e.stopPropagation()}
+        >
+          {item.label}
+          <Dropdown
+            className="theme-dropdown-native-inline"
+            value={year}
+            options={landsatYears}
+            native
+            onChange={this.onLansatChange}
+          />
+        </span>
+      </button>
+    );
+  }
+
+  render() {
+    const {
+      onClose,
+      activeLabels,
+      activeBasemap,
+      getTooltipContentProps,
+      activeBoundaries,
+      selectBoundaries
+    } = this.props;
+    const { boundaries } = this.state;
+    const selectedBoundaries = activeBoundaries
+      ? { label: activeBoundaries.name }
+      : boundaries[0];
+    return (
+      <div className="c-basemaps" {...getTooltipContentProps()}>
+        <div className="basemaps-top-section">
+          <div className="basemaps-header">
+            <h2 className="basemaps-title">Basemap Options</h2>
+            <div className="basemaps-actions">
+              {false && (
+                <button className="basemaps-action-button">
+                  <Icon icon={infoIcon} />
+                </button>
+              )}
+              <button className="basemaps-action-button" onClick={onClose}>
+                <Icon icon={closeIcon} />
+              </button>
+            </div>
+          </div>
+          <ul className="basemaps-options-container">
+            <li className="basemaps-options-wrapper">
+              <Dropdown
+                className="theme-dropdown-button"
+                label="boundaries"
+                value={selectedBoundaries}
+                options={boundaries}
+                onChange={selectBoundaries}
+              />
+            </li>
+            <li className="basemaps-options-wrapper">
+              <Dropdown
+                className="theme-dropdown-button"
+                label="labels"
+                value={activeLabels}
+                options={Object.values(labels)}
```

Comment:
I believe this is the dropdown causing the error with react keys

Reply:
fixed